apps/cli/src/cli/utils.ts

```diff
@@ -116,12 +135,11 @@ export function clirun(
     try {
       await func(useSession, ...args);
     } catch (error) {
-      if (opts.debug === true) {
+      if (opts.debug) {
         useSession.log.debug(`\n\n${(error as Error)?.stack}\n\n`);
-      } else if (opts.debug) {
-        fs.writeFileSync(opts.debug, (error as Error)?.stack ?? '');
```

Comment:
Is debug just always a bool now, no more debug file? I think that's totally fine, but we should change the type if so: https://github.com/curvenote/curvenote/pull/289/files#diff-fb4326bf2e3340b6f868e0fc9fea054853acd5de69849d37ec72518297cfeaffR67